modules/block/factory/go.work

```diff
@@ -0,0 +1,5 @@
+go 1.23
+
+use (
+	.
+)
```

Comment:
You are importing the lakefs package to the factory module - you need to define it in the work file so it will use the local copy
```suggestion
use (
	.
	../../
)
```